code/part-two/README.md

```diff
@@ -77,28 +77,28 @@ source code and some prepackaged Sawtooth components downloaded from
 | ------------- | --------------------- | --------- | ------------------------
 | validator     | tcp://localhost:4004  | DockerHub | Validates blocks and transactions
 | rest-api      | http://localhost:8008 | DockerHub | Provides blockchain via HTTP/JSON
-| processor     | --                    | custom    | The core smart contract logic for your app
-| client        | http://localhost:3000 | custom    | Your front-end, served from `client/public/`
+| processor     | --                    | custom    | Your transaction processor (the core smart contract logic for your app)
```

Comment:
I would drop this change. It's overly wordy for a quick reference table, and the term transaction processor is defined elsewhere).